micropub.php

```diff
@@ -114,17 +110,94 @@ public static function parse_query( $wp ) {
 		else {
 			$user_id = Micropub::authorize();
 		}
+		// If q is set this must be a query so 
```

Comment:
minor nit: "so..."? alternatively, this code is self explanatory, so consider just removing this comment and the one on line 118 altogether. commenting is like goldilocks, you want just enough! :P (also, nit, trailing whitespace.)